packages/core/tables/aggregate-store.ts

```diff
@@ -0,0 +1,13 @@
+import * as AggregateAPI from '@web3-storage/aggregate-api'
+
+export function createAggregateStore() {
+  return useAggregateStore()
+}
+
+export function useAggregateStore() {
+  return {
+    get: async (commitmentProof) => {
```

Comment:
I totally agree. We need to refactor this at the API code level and not here (and in reality for `access-api` and `upload-api`). Would be actually good to do it across the board to be consistent rather than here separately. I created issue for it https://github.com/web3-storage/w3up/issues/818